.github/workflows/prepare-release-pr.yml

```
@@ -34,7 +34,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v5
       with:
-        python-version: "3.8"
+        python-version: "3.13"
```

Alternatively we can use this to mean "latest stable version":

```suggestion
        python-version: "3.x"
```

---

Wow, I didn't know, nice !